src/extensions/vb/Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic/EnableMyDotSupportSubStep.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic
+{
+    /// <summary>
+    /// This analyzer will add &lt;VBRuntime&gt;Embed&lt;/VBRuntime&gt; to the vbproj to resolve compilation errors.
```

Comment:
nit: I believe you can use `<c></c>` to enclose code sections. not sure if it preserves html tags

Reply:
that did not preserve the XML format. The message didn't show the "VBRuntime" portion and only showed the word "Embed".